HandleNHDeleteInstallationCallActivity/handler.ts

```diff
@@ -1,33 +1,30 @@
-import * as t from "io-ts";
-
 import { Context } from "@azure/functions";
 import { toString } from "fp-ts/lib/function";
+import { fromEither } from "fp-ts/lib/TaskEither";
+import * as t from "io-ts";
 
 import { readableReport } from "italia-ts-commons/lib/reporters";
-
-import { fromEither } from "fp-ts/lib/TaskEither";
-import { DeleteInstallationMessage } from "../generated/notifications/DeleteInstallationMessage";
-import { deleteInstallation } from "../utils/notification";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
 
 import {
   ActivityResult,
   ActivityResultFailure,
   ActivityResultSuccess,
   success
 } from "../utils/activity";
+import { deleteInstallation } from "../utils/notification";
 import {
   buildNHService,
   NotificationHubConfig
 } from "../utils/notificationhubServicePartition";
 
 // Activity input
+export type ActivityInput = t.TypeOf<typeof ActivityInput>;
 export const ActivityInput = t.interface({
-  message: DeleteInstallationMessage,
+  installationId: NonEmptyString,
```

Comment:
The activity doesn't need to know the concept of message - it only needs the installationId. Other than semantically correct, it will also enable this activity to be called from other user flows